scm-ui/e2e-tests/cypress.json

```diff
@@ -0,0 +1,5 @@
+{
+  "env": {
+    "BASE_URL": "localhost:8081/scm"
```

Comment:
Why not se baseurl configuration from cypress instead of an environment variable?

https://docs.cypress.io/guides/references/configuration.html#Global